### PR TITLE
AWS CLI (v2) instructions

### PR DESCRIPTION
**Pull request by dled**
It looks like reviewers may be limited to `teamookla/ookla-for-good` at the moment, but I'll send @jj-ookla a RR to double-check our AWS S3 parlance.

Slight tweaks to the S3 access section, and additional instructions for downloading our public S3 objects via CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,14 +82,18 @@ The data is also available in Apache Parquet format. The available fields are de
 
 ### Access via S3
 
-The performance datasets can be accessed on Amazon Web Services (AWS) object storage service, S3, at the s3://ookla-open-data bucket, organized by the key names:
+The performance datasets are available via Amazon Simple Storage Service (Amazon S3) at the `s3://ookla-open-data` bucket, where individual Parquets and compressed Shapefiles are organized by
```
Comment (jj-ookla) [inline README.md:85]:
```suggestion
The performance datasets are available via [AWS S3](https://aws.amazon.com/s3/) at the `s3://ookla-open-data` bucket, where individual [Parquet](https://parquet.apache.org/) time series and compressed Shapefiles are organized by
```

Is there a good link to use for shapefile reference?

File: README.md [hunk 2]
```diff
@@ -82,14 +82,18 @@ The data is also available in Apache Parquet format. The available fields are de
...
 
-1. file format (shapefiles or parquet)
-2. service type (fixed or mobile)
-3. year (2020)
-4. quarter (1 corresponds to the period starting 2020-01-01)
+1. file format (`shapefiles` or `parquet`)
+2. service type (`fixed` or `mobile`)
+3. year (`2020`)
+4. quarter (for example, `1` corresponds to the period starting `2020-01-01`)
```
Comment (jj-ookla) [inline README.md:90]:
```suggestion
4. quarter (for example, `1` corresponds to the Q1 period starting `2020-01-01`)
```

File: README.md [hunk 3]
```diff
@@ -101,21 +105,55 @@ parquet files:
 * `s3://ookla-open-data/parquet/performance/type=mobile/year=2020/quarter=1/2020-01-01_performance_mobile_tiles.parquet`
 * `s3://ookla-open-data/parquet/performance/type=fixed/year=2020/quarter=1/2020-01-01_performance_fixed_tiles.parquet`
 
+Using the key names for the file format, type, year, quarter, and data layer, one can specify the desired dataset.
```
Comment (jj-ookla) [inline README.md:108]:
```suggestion
```

I think this is explained pretty well above with the numbered list and example(s).

File: README.md [hunk 4]
```diff
@@ -101,21 +105,55 @@ parquet files:
...
 
 * https://ookla-open-data.s3.amazonaws.com/parquet/performance/type=mobile/year=2020/quarter=1/2020-01-01_performance_mobile_tiles.parquet
 * https://ookla-open-data.s3.amazonaws.com/parquet/performance/type=fixed/year=2020/quarter=1/2020-01-01_performance_fixed_tiles.parquet
 
-using the key names for the file format, type, year, quarter, and data layer, as above.
+### Download via CLI
+
+S3 objects can also be downloaded via the AWS CLI. See these instructions for [installing AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).  
+
+Based on the object key names described above, the following bash script downloads compressed shapefiles for fixed performance tiles aggregated over the first calendar year quarter of 2020 using the [aws s3 cp](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/cp.html) command:
```
Comment (jj-ookla) [inline README.md:128]:
```suggestion
Using the object keys described above, the following bash script downloads a shapefile (`2020-01-01_performance_fixed_tiles.zip`) for fixed performance tiles aggregated over Q1 2020 using [aws s3 cp](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/cp.html).
```

File: README.md [hunk 5]
```diff
@@ -101,21 +105,55 @@ parquet files:
...
+export TYPE='fixed'        # (fixed|mobile)
+export YYYY='2020'         # 2020
+export Q='1'               # 1,2,3 (to date)
+
+aws s3 cp s3://ookla-open-data/${FORMAT}/performance/type=${TYPE}/year=${YYYY}/quarter=${Q}/ . \
+--recursive \
+--no-sign-request
+```
+
+In this example, we pass the `--recursive` option to the AWS S3 copy command to download the compressed shapefiles located at the S3 URI defined by service type, year, and quarter. The filename corresponds to the start date of the quarter and the network service type: `2020-01-01_performance_fixed_tiles.zip`.
```
Comment (jj-ookla) [inline README.md:142]:
```suggestion
```

This seems redundant. I edited the text above to include the expected filename.

File: README.md [hunk 6]
```diff
@@ -101,21 +105,55 @@ parquet files:
...
+
+To download the mobile and fixed datasets in parquet format for 2020 Q3, we can also specify the full S3 URI path to download the objects:
```
Comment (jj-ookla) [inline README.md:144]:
```suggestion
To download the 2020 Q3 mobile and fixed time series datasets, we can also specify the full S3 URI to download the objects:
```

File: README.md [hunk 7]
```diff
@@ -101,21 +105,55 @@ parquet files:
...
+#!/usr/bin/env bash
+
+# Mobile 2020 Q3
+aws s3 cp s3://ookla-open-data/parquet/performance/type=mobile/year=2020/quarter=3/2020-07-01_performance_mobile_tiles.parquet --no-sign-request
+
+# Fixed 2020 Q3
+aws s3 cp s3://ookla-open-data/parquet/performance/type=fixed/year=2020/quarter=3/2020-07-01_performance_fixed_tiles.parquet --no-sign-request
+```
+
+AWS accounts within an organization may have restrictions on accessing external resources via CLI. We've therefore included the  `--no-sign-request` flag in the examples above, which allows us to copy S3 objects locally using public requests.
```
Comment (jj-ookla) [inline README.md:156]:
```suggestion
```

**Review by jj-ookla — APPROVED** [review]
LGTM!

I trimmed down some of the sections to keep the focus on example commands.

**Review by smpetersen — COMMENTED** [review]
Late to the party, but this is great. Will help a lot of people out!

**Pull request merged**